### PR TITLE
Tracy v0.5.5

### PR DESCRIPTION
**Pull request by tobiasrausch**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/tracy/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,41 @@
...
   version: '{{ version }}'
 
 source:
-  url: https://github.com/gear-genomics/tracy/archive/v{{ version }}.tar.gz
-  sha256: '{{ sha256 }}'
+  - url: https://github.com/gear-genomics/tracy/archive/v{{ version }}.tar.gz
+    sha256: '{{ sha256 }}'
+  - folder: src/xxsds
+    git_url: https://github.com/xxsds/sdsl-lite.git
+    git_rev: a654b57
```
Comment (dpryan79) [inline recipes/tracy/meta.yaml]: Does the sdsl-lite package not work for this?

**Comment by dpryan79** [comment]
If you had a good reason for using sdsl from github rather than the sdsl-list package then go ahead and merge.

File: recipes/tracy/meta.yaml (hunk 1 above)
Comment (tobiasrausch) [reply recipes/tracy/meta.yaml]: Sdsl-lite is GPL, SDSL v3 (github) is BSD so we could make tracy BSD licensed as well.

Comment (dpryan79) [reply recipes/tracy/meta.yaml]: Ah, gotcha, makes sense. :+1:

**Pull request merged**